rails6/en/chapter02-api.adoc

```diff
@@ -15,34 +15,34 @@ To summarize, we simply generated our Rails application and made our first commi
 
 == Planning the application
 
-As we want to go simple with the application it consists on five models. Don’t worry if you don’t fully understand what is going on. We will review and build each of these resources as we move on with the tutorial.
+As we want to go simple with the application it consists of five models. Don’t worry if you don’t fully understand what is going on. We will review and build each of these resources as we move on with the tutorial.
 
 image:data_model.png[Schema of links betweens models]
 
-In brief, the `user` will be able to place many `orders`, upload multiple `products` which can have many `images` or `comments` from another users on the app.
+In brief, the `user` will be able to place many `orders`, upload multiple `products` which can have many `images` or `comments` from other users on the app.
 
 We are not going to build views for displaying or interacting with the API, so not to make this a huge tutorial, I’ll let that to you. There are plenty of options out there like javascript frameworks (https://angularjs.org/[Angular], https://vuejs.org/[Vue.js], https://reactjs.org/[React.js]).
 
 By this point you must be asking yourself:
 
-> all right but I need to explore or visualize the api we are going to be building?
+> all right but I need to explore or visualize the API we are going to be building?
 
-That’s fair. Probably if you google something related to api exploring, an application called https://www.getpostman.com/[Postman] will pop. It is a great software but we won’t be using that anyway because we'll use *cURL* who allow anybody to reproduce request on any computer.
+That’s fair. Probably if you google something related to API exploring, an application called https://www.getpostman.com/[Postman] will pop. It is great software but we won’t be using that anyway because we'll use *cURL* which allows anybody to reproduce requests on their computer.
```

Comment:
Isn't "any" make more sens ?

```suggestion
That’s fair. Probably if you google something related to API exploring, an application called https://www.getpostman.com/[Postman] will pop. It is great software but we won’t be using that anyway because we'll use *cURL* which allows anybody to reproduce requests on any computer.
```

Reply:
Yes sure